synapse/handlers/room.py

```diff
@@ -1526,6 +1526,8 @@ async def get_event_for_timestamp(
                     await self.store.is_event_next_to_forward_gap(local_event)
                 )
 
+        likely_domains = None
```

Comment:
TODO: We can remove this when we remove the local `outlier` backfill below as well: https://github.com/matrix-org/synapse/pull/14096#discussion_r997710667

Reply:
Removed ✅